tests/lambda_endpoints/conftest.py

```diff
@@ -33,6 +33,7 @@ def _set_env(monkeypatch):
     # Mock the api gateway url
     # This is used in batch_starter.py
     monkeypatch.setenv("IMAP_DATA_ACCESS_URL", "https://test.url")
+    monkeypatch.setenv("EFS_SPICE_MOUNT_PATH", str(tmpdir))
```

Comment:
IMAP_DATA_DIR of which the SPICEFilePath is relative to? I think this can be a follow-up PR now that I'm reading it closer. But it would be nice to not have these mutliple different ways to point to different paths.